chat-plugins/info.js

```diff
@@ -1152,46 +1152,50 @@ exports.commands = {
-			"More detailed help can be found <a href=\"https://gist.github.com/verbiage/0846a552595349032fbe\">here</a><br />" +
-			"</div>"
+			"<details>" +
+				"<summary>Room Auth Commands:</summary>" +
+				"Room drivers (%) can use:<br />" +
+				"- /warn OR /k <em>username</em>: warn a user and show the Pok&eacute;mon Showdown rules</p><br />" +
+				"- /mute OR /m <em>username</em>: 7 minute mute<br />" +
+				"- /hourmute OR /hm <em>username</em>: 60 minute mute<br />" +
+				"- /unmute <em>username</em>: unmute<br />" +
+				"- /announce OR /wall <em>message</em>: make an announcement<br />" +
+				"- /modlog <em>username</em>: search the moderator log of the room<br />" +
+				"- /modnote <em>note</em>: adds a moderator note that can be read through modlog<br />" +
+				"<br />" +
+				"Room moderators (@) can also use:<br />" +
+				"- /roomban OR /rb <em>username</em>: bans user from the room<br />" +
+				"- /roomunban <em>username</em>: unbans user from the room<br />" +
+				"- /roomvoice <em>username</em>: appoint a room voice<br />" +
+				"- /roomdevoice <em>username</em>: remove a room voice<br />" +
+				"- /modchat <em>[off/autoconfirmed/+]</em>: set modchat level<br />" +
+				"- /staffintro <em>intro</em>: sets the staff introduction that will be displayed for all staff joining the room<br />" +
+				"<br />" +
+				"Room owners (#) can also use:<br />" +
+				"- /roomintro <em>intro</em>: sets the room introduction that will be displayed for all users joining the room<br />" +
+				"- /rules <em>rules link</em>: set the room rules link seen when using /rules<br />" +
+				"- /roommod, /roomdriver <em>username</em>: appoint a room moderator/driver<br />" +
+				"- /roomdemod, /roomdedriver <em>username</em>: remove a room moderator/driver<br />" +
+				"- /roomdeauth <em>username</em>: remove all room auth from a user<br />" +
+				"- /modchat <em>[%/@/#]</em>: set modchat level<br />" +
+				"- /declare <em>message</em>: make a large blue declaration to the room<br />" +
+				"- !htmlbox <em>HTML code</em>: broadcasts a box of HTML code to the room<br />" +
+				"- !showimage <em>[url], [width], [height]</em>: shows an image to the room<br />" +
+				"<br />" +
+				"More detailed help can be found in the <a href=\"https://www.smogon.com/forums/threads/3570628/#post-6774654\">roomauth guide</a><br />" +
+				"<br />" +
+			"</details>" +
+			"<details>" +
+				"<summary>Tournament Help:</summary>" +
+				"- /tour create <em>format</em>, elimination: Creates a new single elimination tournament in the current room.<br />" +
+				"- /tour create <em>format</em>, roundrobin: Creates a new round robin tournament in the current room.<br />" +
+				"- /tour end: Forcibly ends the tournament in the current room<br />" +
+				"- /tour start: Starts the tournament in the current room<br />" +
+				"<br />" +
+				"More detailed help can be found <a href=\"https://gist.github.com/verbiage/0846a552595349032fbe\">here</a><br />" +
```

Comment:
This should link to new guide: https://www.smogon.com/forums/threads/3570628/#post-6777489